app/src/main/kotlin/com/infinity/foodstandards/ui/authorities/AuthoritiesViewModel.kt

```diff
@@ -7,16 +7,18 @@ import androidx.lifecycle.viewModelScope
 import com.infinity.foodstandards.model.LocalAuthoritiesResponse
 import com.infinity.foodstandards.network.FoodStandardsRepo
 import com.infinity.foodstandards.network.RetrofitService
-import kotlinx.coroutines.CoroutineDispatcher
-import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 
 class AuthoritiesViewModel() : ViewModel() {
 
     private var foodStandardsRepo: FoodStandardsRepo =
         FoodStandardsRepo(RetrofitService.createService())
 
-    private val localAuthorities = MutableLiveData<LocalAuthoritiesResponse>()
+     val localAuthorities = MutableLiveData<LocalAuthoritiesResponse>()
+
+    init {
```

Comment:
Instead of running this on the view model init function, I think it should be ran from the LaunchedEffect in the composable like below, what do you think?

```
setContent {
    LaunchedEffect(Unit) { // Use unit as we just want this as a one time call
        viewModel.getLocalAuthorities()
    }
}
```

Reply:
Thanks. Will update this

Reply:
Probably didn't need the comment that was just me explaining why I used Unit there 😂 but fix looks good 👍